addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -290,7 +290,485 @@
             t-att-data-variable="radius_variable"/>
 </template>
 
-<template id="snippet_options_extra_shapes" inherit_id="web_editor.snippet_options">
+<template id="snippet_options_background_options" inherit_id="web_editor.snippet_options_background_options" primary="True">
+    <xpath expr="//div[@data-js='BackgroundImage']" position="attributes">
+        <attribute name="data-js">BackgroundImageMassMailing</attribute>
+    </xpath>
+</template>
+<!-- Snippet themes Options -->
+<template id="snippet_options" inherit_id="web_editor.snippet_options" primary="True">
+    <!-- t-set -->
+    <xpath expr="//t[@t-set='no_animations']" position="attributes">
+        <attribute name="t-value">True</attribute>
+    </xpath>
+
+    <!-- Transform is _very_ badly supported in mail clients -->
+    <xpath expr="//we-button[@data-name='image_transform_opt']" position="replace" />
+
+    <xpath expr="//div[@data-js='ImageTools']" position="attributes">
+        <attribute name="data-js">ImageToolsMassMailing</attribute>
+    </xpath>
+
+    <!-- Adding new Snippet Options -->
+
+    <xpath expr="." position="inside">
```

Comment:
Using an external template like "extra_snippet_options" means having to change every options added by specific snippet (grep "website.snippet_options") I don't mind it but it will add a bigger diff and IMO might make it harder to understand which template to target when trying to extend/modify options from an external module.

For now I went with the solution of undoing the indent I did, what do you think?